src/screens/admin/AllEmployers.js

```diff
@@ -29,6 +48,110 @@ export default class AllEmployers extends React.Component {
     }
     
   }
+
+  handleFormChange(field, value) {
+    this.setState({[field]: value});
+  }
+
+  handleSignUp = async () => {
+    const { email, password, name, company, phone, address, confirmPassword} = this.state;
+    if (!email || !password || !name || !company || !phone || !address || !confirmPassword) {
+      alert("Please fill in all required fields");
+    } else {
+      try {
+        const apiCreateCompany = await fetch(`${API.endpoint}/companies`, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            name: company,
+            address,
+          }),
+        });
+        const createdCompany = await apiCreateCompany.json();
+        const companyId = createdCompany.insertId.toString();
+        if (password === confirmPassword) {
+          try {
+            await Auth.signUp({
+              username: email,
+              password,
+              attributes: {
+                email, 
+                name, 
+                'phone_number': phone, 
+                'profile': companyId,
+              },
+            })
+            Alert.alert(
+              "Employer sign-up pending confirmation.",
+              "A verification email has been sent to the Employer. They may sign in after clicking the verification link.", 
+              [{title: "OK"}]
+            )
+          } catch (err) {
+            console.log(err);
+            alert(err.message);
+            try {
+              await fetch(`${API.endpoint}/companies/${companyId}`, {
+                method: 'DELETE'
+              });
+            } catch (err) {
+              console.log(err)
+            }
+          }
+        } else {
+          alert('Passwords do not match.');
+        }
+      } catch (err) {
+        console.log(err);
+        Alert.alert(
+          "Employer sign-up failed.",
+          err,
+          [{title: "OK"}]
+        )
+      }
+    }
+  };
+
+  onCancelAdd = () => {
+    this._panelUsers.hide();
+  };
+
+  onDoneAdd = async () => {
+    this._panelUsers.hide();
+    await this.handleSignUp();
+  }
+
+  renderContent = () => {
+    return (
+      <View style={styles.container}>
+        <Header
+          backgroundColor="#FFF"
+          leftComponent={
+            <TouchableHighlight
+              underlayColor="#FFF"
+              onPress={() => this.onCancelAdd()}
+            >
+              <Text style={styles.headerText}>Cancel</Text>
+            </TouchableHighlight>
+          }
+          rightComponent={
+            <TouchableHighlight
+              underlayColor="rgba(255, 255, 255, 0.92)"
+              onPress={() => this.onDoneAdd()}
+            >
+              <Text style={styles.headerText}>Create</Text>
+            </TouchableHighlight>
+          }
+        />
+        <SafeAreaView style={{flex: 1}}>
```

Comment:
Make the signup page scrollable because when we reach the phone number and address sections the keyboard covers the fields and we can't see what we're typing. (Note: not sure if the change is in SafeAreaView)